### PR TITLE
fix: pass contextIsolation: true

### PR DESCRIPTION
**Pull request by flotwig**
- Closes 

### Additional details

- https://github.com/cypress-io/cypress/pull/16113 probably introduced this 
- was never released, but fixes this error on desktop-gui launch: 
image

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -15960,10 +15960,10 @@ electron-to-chromium@^1.3.247, electron-to-chromium@^1.3.378, electron-to-chromi
...
   integrity sha512-3kRVibBeCM4vsgoHHGKHmPocLqtFAGTrebXxxtgKs87hNUzXrX2NuS3jnBys7IozCnw7viQlozxKkmty2KNfrw==
 
-electron@12.0.2:
-  version "12.0.2"
-  resolved "https://registry.yarnpkg.com/electron/-/electron-12.0.2.tgz#d92be205f1937627bd6718aad44ac161382b4c2d"
-  integrity sha512-14luh9mGzfL4e0sncyy0+kW37IU7Y0Y1tvI97FDRSW0ZBQxi5cmAwSs5dmPmNBFBIGtzkaGaEB01j9RjZuCmow==
+electron@12.0.0-beta.14:
+  version "12.0.0-beta.14"
+  resolved "https://registry.npmjs.org/electron/-/electron-12.0.0-beta.14.tgz#f8c40c7e479879c305e519380e710c0a357aa734"
+  integrity sha512-PYM+EepIEj9kLePXEb9gIxzZk5H4zM7LGg5iw60OHt+SYEECPNFJmPj3N6oHKu3W+KrCG7285Vgz2ZCp1u0kKA==
```
Comment (Bkucera) [inline yarn.lock:15966]: purposeful?

Comment (flotwig) [reply yarn.lock:15966]: yes

**Review by Bkucera — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (Bkucera) [reply yarn.lock:15966]: does our binary build script use yarn --frozen-lockfile? is there no package.json change to go along with this

Comment (flotwig) [reply yarn.lock:15966]: apparently not anymore, it used to

Comment (flotwig) [reply yarn.lock:15966]: the package.json change was in the downgrade PR

**Pull request merged**